cmake/sdp.cmake

```diff
@@ -4,6 +4,24 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+function(sdp_assembly hrt_srcs)
```

Comment:
please describe the function and its arguments.

And would be nice if you could also followup on the same for other functions, as that seems to have been overlooked in #17638